sopel/modules/isup.py

```diff
@@ -60,22 +60,36 @@ def handle_isup(bot, trigger, secure=True):
     """
     try:
         site = get_site_url(trigger.group(2))
```

Comment:
If you can check that argument, that would do it for #1001 !

Reply:
Doesn't this snippet of `get_site_url()` already do that?

https://github.com/sopel-irc/sopel/blob/50ace55f2b6309b59792aeda332e12ed79472a16/sopel/modules/isup.py#L32-L34

I guess it could be checked _in_ `handle_isup()` instead, but that validation function is already covered by unit tests (including empty `site` arg, though `None` wasn't tested; I just added that).

Reply:
Great!